tests/foreman/ui/test_host.py

```diff
@@ -1878,6 +1879,45 @@ def test_all_hosts_bulk_delete(target_sat, function_org, function_location, new_
         assert session.all_hosts.bulk_delete_all()
 
 
+@pytest.mark.tier2
+def test_all_hosts_bulk_cve_reassign(
+    target_sat, mod_content_hosts, module_org, module_location, module_lce, module_cv, new_host_ui
+):
+    """Create several hosts, and bulk assigns them a new CVE via All Hosts UI
+
+    :id: 9acb3cf3-c042-4cc7-abdf-31f9a7f1fff0
+
+    :expectedresults: Multiple hosts are successfully assigned to a new LCE and CV
+
+    :CaseComponent:Hosts-Content
+
+    :Team: Phoenix-subscriptions
+    """
+    lce2 = target_sat.api.LifecycleEnvironment(organization=module_org).create()
+    module_cv = target_sat.api.ContentView(id=module_cv.id).read()
+    module_cv = cv_publish_promote(target_sat, module_org, module_cv, module_lce)['content-view']
+    module_cv = cv_publish_promote(target_sat, module_org, module_cv, lce2)['content-view']
+    ak = target_sat.api.ActivationKey(
+        organization=module_org,
+        environment=module_lce,
+        content_view=module_cv,
+    ).create()
+    for host in mod_content_hosts:
+        host.register(
+            activation_keys=ak.name,
+            target=target_sat,
+            org=module_org,
+            loc=None,
+        )
```

Comment:
We could use `module_ak_with_cv`  (together with `module_promoted_cv`) so we don't recreate it for every RHEL parametrization. Worked for me locally.
```suggestion
    module_promoted_cv = cv_publish_promote(target_sat, module_org, module_promoted_cv, lce2)['content-view']
    for host in mod_content_hosts:
        host.register(
            activation_keys=module_ak_with_cv.name,
            target=target_sat,
            org=module_org,
            loc=None,
        )
```